build/check-images.ts

```diff
@@ -25,7 +25,8 @@ export function checkImageReferences(
   options,
   { url, rawContent }
 ) {
```

Comment:
Let's make the return type explicit (and also add it to the JSDoc above, something along the lines of `@returns map of full image paths by basename.`):

```suggestion
): Map<string, string> {
```